charts/camunda-platform/values.yaml

```diff
@@ -579,6 +585,7 @@ tasklist:
   # contextPath: "/tasklist"
 
   # Env can be used to set extra environment variables on each Tasklist container
+  # See https://docs.camunda.io/docs/self-managed/tasklist-deployment/configuration/
```

Comment:
Redirects to: https://docs.camunda.io/docs/self-managed/tasklist-deployment/tasklist-configuration/